test/postSkill.js

```diff
@@ -0,0 +1,27 @@
+const test = require('tape');
+const supertest = require('supertest');
+const router = require('../server/app');
+
+test('Testing /api/v1/skills  route', (t) => {
+  const fields = { id: 4, name: 'JAVAFX' };
+  const body = {
+    name: 'JAVAFX',
+  };
+
+  supertest(router)
+    .post('/api/v1/skills')
+    .send(body)
+    .expect(200)
+    .end((err, result) => {
+      if (err) {
+        t.error(err);
+        t.end();
+      }
+      t.deepEqual(result.body.data, fields, 'Should contain the same content');
```

Comment:
@Angham116 it might make life easier for you if you only check that result.body.data.name equals JAVAFX, and ignore the value of ID. Then if your dB fake data changes, causing the inserted id in this test to change, the test won’t fail. Ultimately, the name is what is important to test so I think that would be a safe change. 

Also, it would be awesome to add a test which ensures that the endpoint returns an error when you try to add a duplicate skill name. If you have time later in the project :)

Reply:
@tomduggan85  I have thought about it yesterday & I will do it, thanx

Reply:
@Angham116 thanks for doing that!